server.py

```diff
@@ -175,6 +188,61 @@ def statistics():
     )
 
 
+@app.get("/addQuestion")
+@login_required
+def addQuestion():
+    return bottle.template("addQuestion.html")
+
+
+@app.post("/questionInput")
+@login_required
+def questionInput():
+    userid = Session.get(Session.token == bottle.request.get_cookie("s_id")).user
+    time = datetime.datetime.now()
+    uploaded_question = bottle.request.files.get("question").file.read()
+    uploaded_answer = bottle.request.files.get("answer").file.read()
+    uploaded_statement = bottle.request.forms.get("statement")
+    try:
+        UploadedQuestion.create(
+            question_text=uploaded_question,
+            answer_text=uploaded_answer,
+            statement_text=uploaded_statement,
+            author=userid,
+            created_date_time=time,
+        )
+    except:
+        bottle.abort(500, "Error in inserting submission to database.")
+    question_bank = (
+        UploadedQuestion.select(
+            UploadedQuestion.id,
+            UploadedQuestion.question_text,
+            UploadedQuestion.statement_text,
+            User.username,
+            UploadedQuestion.created_date_time,
+        )
+        .join(User, on=(UploadedQuestion.author == User.id))
+        .order_by(UploadedQuestion.created_date_time.desc())
+        .dicts()
+    )
+    return bottle.template("questionBank.html", question_bank=question_bank)
+
+
+@app.get("/display/<id>")
+@app.post("/display/<id>")
+@login_required
+def displayQuestion(id):
```

Comment:
ah. you'll have to replace the contest question url instead of creating a new one.